accord-core/src/main/java/accord/coordinate/FetchMaxConflict.java

```diff
@@ -85,7 +85,7 @@ public static AsyncResult<Timestamp> fetchMaxConflict(Node node, Seekables<?, ?>
     @Override
     void contact(Set<Node.Id> nodes, Topologies topologies, Callback<GetMaxConflictOk> callback)
     {
-        node.send(nodes, to -> new GetMaxConflict(to, topologies, route, keysOrRanges, executionEpoch));
+        node.send(nodes, to -> new GetMaxConflict(to, topologies, route, keysOrRanges, executionEpoch), callback);
```

Comment:
the fact this happened implies that burn test doesn't hit this code path, so should be something we think about adding... C* hits this all the time as we don't have a max applied, so we would block forever